notebooks/.gitignore

```diff
@@ -0,0 +1 @@
+.ipynb_checkpoints*
```

Comment:
This can be added to the top-level .gitignore rather than introducing a second one.